wp_api/tests/wpcom/subscribers/individual-subscriber-stats.json

```diff
@@ -0,0 +1,6 @@
+{
+  "emails_sent": 2,
+  "unique_opens": 1,
+  "unique_clicks": 2,
+  "blog_registration_date": "2025-01-21 12:14:50"
```

Comment:
`WpGmtDateTime` is used to parse this date. Are we sure this date string is in the UTC timezone?

Reply:
Good catch! Updated to be `String` in 33c6bb19da2efe5d730dab5301e5e091126f1f2a.